ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent-nonprod.json

```diff
@@ -62,27 +62,6 @@
     "UserRole": "[LASOLICITOR]",
     "CRUD": "CRU"
   },
-  {
-    "LiveFrom": "01/01/2017",
-    "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "CaseEventID": "placement",
-    "UserRole": "caseworker-publiclaw-solicitor",
-    "CRUD": "CRU"
-  },
-  {
-    "LiveFrom": "01/01/2017",
-    "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "CaseEventID": "placementGatekeeping",
-    "UserRole": "caseworker-publiclaw-solicitor",
-    "CRUD": "CRU"
-  },
-  {
-    "LiveFrom": "01/01/2017",
-    "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "CaseEventID": "placement-PREPARE_FOR_HEARING",
-    "UserRole": "caseworker-publiclaw-solicitor",
-    "CRUD": "CRU"
-  },
```

Comment:
why are we removing placement permission here? intended?

Reply:
yes, it is intended. Events should  [LASOLICITOR] and not caseworker-publiclaw-solicitor. LASOLICITOR is already configured there

Reply:
Makes sense, thank you